core/src/test/java/org/aerogear/mobile/core/MobileCoreTest.java

```diff
@@ -26,12 +28,25 @@
 @SmallTest
 public class MobileCoreTest {
 
-    @Test
-    public void testInit() {
+    private static final String DUMMY_MOBILE_SERVICES_JSON = "dummy-mobile-services.json";
+    private static final String EMPTY_MOBILE_SERVICES_JSON = "empty-mobile-services.json";
+
+    private MobileCore core;
+    private MobileCore dummyCore;
```

Comment:
why `core` and `dummyCore`?

Reply:
Core is using default config file with the default services (the production-like version if you will) and dummycore is for testing weird cases, dummy services and so on, it has a custom config file made only for testing.